src/components/LoginForm.tsx

```diff
@@ -0,0 +1,129 @@
+import styled from "@emotion/styled";
+import type { InputErrorHandlerHook } from "@src/hooks/useInputErrorHandler";
+import useInputErrorHandler from "@src/hooks/useInputErrorHandler";
+import useUserInfo, { type UserInfoHook } from "@src/hooks/useUserInfo";
+import theme from "@src/styles/kakaoTheme";
+import { createNewIDEvaluator } from "@src/utils/evaluator/implementation/idEvaluator";
+import { createNewPWEvaluator } from "@src/utils/evaluator/implementation/passwordEvaluator";
+import { useNavigate } from "react-router-dom";
+
+function LoginForm() {
+  const navigate = useNavigate();
+  const userInfo = useUserInfo();
+
+  const idEvaluator = createNewIDEvaluator();
+  const pwEvaluator = createNewPWEvaluator();
+  const inputErrorHandler: InputErrorHandlerHook = useInputErrorHandler();
+
+  const handleLogin = (userInfo: UserInfoHook) => {
```

Comment:
이미 함수 밖에서 `userInfo`를 받았는데 또 매개변수로 받을 필요는 없을 것 같아요!
```suggestion
  const handleLogin = () => {
```